cp1616/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(cp1616)
+
+find_package(catkin REQUIRED COMPONENTS 
+  roscpp
+  message_generation
+  )
+
+include_directories(${catkin_INCLUDE_DIRS})
+include_directories(${PROJECT_SOURCE_DIR}/include/)
+
+
+SET(SOURCES ${PROJECT_SOURCE_DIR}/src/cp1616_io_controller.cpp
+            ${PROJECT_SOURCE_DIR}/src/cp1616_io_controller_callbacks.cpp
+            ${PROJECT_SOURCE_DIR}/src/cp1616_io_device.cpp
+            ${PROJECT_SOURCE_DIR}/src/cp1616_io_device_callbacks.cpp
+            ${PROJECT_SOURCE_DIR}/src/cp1616_main.cpp
+          )
+
+SET(HEADERS ${PROJECT_SOURCE_DIR}/include/cp1616/cp1616_io_controller.h
+            ${PROJECT_SOURCE_DIR}/include/cp1616/cp1616_io_controller_callbacks.h
+            ${PROJECT_SOURCE_DIR}/include/cp1616/cp1616_io_device.h
+            ${PROJECT_SOURCE_DIR}/include/cp1616/cp1616_io_device_callbacks.h
+	  )
+	  
+catkin_package(
+  DEPENDS
+  INCLUDE_DIRS
+  CATKIN_DEPENDS roscpp 
+  LIBRARIES
+  roscpp
```

Comment:
You are right, I suggest this:

catkin_package(
  INCLUDE_DIRS include
  LIBRARIES Cp1616RosLib
  CATKIN_DEPENDS roscpp 
  DEPENDS libpnoiusr libl2interface 
)